masterlist.yaml

```diff
@@ -12858,13 +12858,12 @@ plugins:
   - name: 'Imperial Shipyard.esp'
     url: [ 'https://www.nexusmods.com/oblivion/mods/51763/' ]
     req: [ 'DLCShiveringIsles.esp' ]
-    inc: [ 'LetThePeopleDrink.esp' ]
     dirty:
       - <<: *quickClean
-        crc: 0x33EE0585
+        crc: 0x35FD4202
         util: '[TES4Edit v4.0.4b](https://www.nexusmods.com/oblivion/mods/11536)'
-        itm: 5245
-        udr: 13
+        itm: 5222
+        udr: 52
```

Comment:
V0.5 was released yesterday & appears to have been run through QAC. However, there appears to be at least one wild edit (NPC `0018B118`), so a message could be added to replace the old cleaning data (sorry, the line deletions are preventing a suggestion), or the old cleaning data can just be removed. The new checksum is 0x09D58350:

```yaml
    msg:
      - <<: *wildEdits
        subs: [ 'Remove NPC **0018B118** from **Imperial Shipyard.esp**.' ]
        condition: 'checksum("Imperial Shipyard.esp", 09D58350)'
```